src/types/value/link.ts

```diff
@@ -33,6 +33,7 @@ export interface FilledLinkToWebField {
 	link_type: typeof LinkType.Web
 	url: string
 	target?: string
+	text?: string
```

Comment:
Same comment as @lihbr here: https://github.com/prismicio/prismic-client/pull/348/files#r1716651942

Reply:
Done ✅

Reply:
Actually, I reverted this back after discussing with @xrutayisire & @dani-mp. We intentionally want to keep this as a string and don't intend for it to behave in the same way as a key text field.